lib/rubocop/cop/lint/out_of_range_regexp_ref.rb

```diff
@@ -19,7 +19,7 @@ module Lint
       #   puts $1 # => foo
       #
       class OutOfRangeRegexpRef < Base
-        MSG = 'Do not use out of range reference for the Regexp.'
+        MSG = '$%<backref>s is out of range (%<count>s Regexp capture groups detected).'
```

Comment:
I'd use `regexp` here without the capital letter. No idea why it was there in the first place.